platform/viewer/src/lib/DICOMSR/retrieveDataFromMeasurements.js

```diff
@@ -0,0 +1,52 @@
+import * as dcmjs from 'dcmjs';
+import OHIF from '@ohif/core';
+import cornerstone from 'cornerstone-core';
+
+const retrieveDataFromMeasurements = measurements => {
```

Comment:
What is the shape of measurements? Can we JSDoc this? (note: it supports a `@typedef`)
Any way to only import the portions of dcmjs we need instead of everything?